assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceTilesWidget/index.test.js

```diff
@@ -248,67 +248,25 @@ describe( 'AudienceTilesWidget', () => {
 
 		await waitForRegistry();
 
-		expect( container ).toBeEmptyDOMElement();
-	} );
-
-	it( 'should not render when availableAudiences is not loaded', async () => {
-		registry.dispatch( CORE_USER ).receiveGetAudienceSettings( {
-			configuredAudiences: [ 'properties/12345/audiences/1' ],
-			isAudienceSegmentationWidgetHidden: false,
-		} );
-
-		const { container, waitForRegistry } = render(
-			<WidgetWithComponentProps />,
-			{
-				registry,
-			}
-		);
-
-		await waitForRegistry();
-
-		expect( container ).toBeEmptyDOMElement();
+		expect(
+			getByText( /You don’t have any visitor groups selected./i )
+		).toBeInTheDocument();
 	} );
 
-	it( 'should not render when configuredAudiences is not loaded', async () => {
-		muteFetch( audienceSettingsRegExp );
-
-		registry
-			.dispatch( MODULES_ANALYTICS_4 )
-			.setAvailableAudiences( availableAudiences );
-
-		const { container, waitForRegistry } = render(
-			<WidgetWithComponentProps />,
-			{
-				registry,
-			}
+	it( 'should render loading tiles when audiences are not syncing', async () => {
+		const maybeSyncAvailableAudiencesSpy = jest.spyOn(
+			registry.dispatch( MODULES_ANALYTICS_4 ),
+			'maybeSyncAvailableAudiences'
 		);
 
-		await waitForRegistry();
-
-		expect( container ).toBeEmptyDOMElement();
-	} );
-
-	it( 'should not render when there is no available audience', async () => {
-		registry.dispatch( MODULES_ANALYTICS_4 ).setAvailableAudiences( [] );
-
-		registry.dispatch( CORE_USER ).receiveGetAudienceSettings( {
-			configuredAudiences: [ 'properties/12345/audiences/9' ],
-			isAudienceSegmentationWidgetHidden: false,
-		} );
-
-		const { container, waitForRegistry } = render(
-			<WidgetWithComponentProps />,
-			{
-				registry,
-			}
+		/**
+		 * Mock the maybeSyncAvailableAudiences function to return a promise that never resolves
+		 * so that availableAudiencesSynced is never set to true.
+		 */
+		maybeSyncAvailableAudiencesSpy.mockImplementation(
+			() => new Promise( () => {} )
 		);
```

Comment:
Rather that mocking the `maybeSyncAvailableAudiences()`, action we can freeze the fetch to the underlying request. This would be more consistent with our general approach to testing:

```suggestion
		const syncAvailableAudiencesEndpoint = new RegExp(
			'^/google-site-kit/v1/modules/analytics-4/data/sync-audiences'
		);

		freezeFetch( syncAvailableAudiencesEndpoint );
```